osdk/src/cli.rs

```diff
@@ -68,10 +69,18 @@ pub enum OsdkSubcommand {
     Run(RunArgs),
     #[command(about = "Execute kernel mode unit test by starting a VMM")]
     Test(TestArgs),
-    #[command(about = "Analyze the current package and report errors")]
-    Check,
-    #[command(about = "Check the current package and catch common mistakes")]
-    Clippy,
+    #[command(about = "Wrapper for the `cargo check` command")]
```

Comment:
`Wrapper` is too implementation oriented. To a user, it is sufficient to know that `cargo osdk check` is equivalent to `cargo check`.